templates/static/accessibility.html

```diff
@@ -119,11 +117,11 @@ <h3>{{ _("Digital Accessibility Centre testing") }}</h3>
 
 
-  <p>{{ _("<a href='{url}' download>DAC Report - September 2019</a>").format(url = "https://00e9e64bac927483db2dfd4487850d3028707017bf2b680f4b-apidata.googleusercontent.com/download/storage/v1/b/census-ci-craftcms/o/assets%2FONS-Census-Survey-2021-DAC-Accessibility-Report-Sept-2019.pdf?qk=AD5uMEv4OOShhFARPG-naoLGLStfFwDhiJI9Ppl2oGD8z5UMt3RTdkZOHQupf4-x6E2MOcwbrnCGFPoWF1WTZp0TdnA3QVU1KYuYt3_hpMKS3zcsd1uQESRvF4qU5cRkq34VMSVMKD4ZcPkvab26rSnAt_8qNykO-1aCuOJxNzf0UxhF19b-Pvpy0B5LnzswSkgQN-OmJS7JNKzQMaBelwaSj44Xkg0WSsK32gPxSpxHhL5Bu8DE2ntc8Dm9ppaI01S3WF8xkrOUmj6dtZAXl2AvCHUc64QOddRJKJDh1CHSSMXH_o7YeccP4JVkHXZx4GpO8an62GZVDsAZaZJfesam7A9II5C8nAGmJyYREzhMjgJtz0zxITFLXOVBdj6g9jTXD8VxfdM00YWH_x7fDVPwNUKcVdCTSXWPAlmX66dyj_BEz23guQqOK5pyrPlVMfZfBlBBTyg_fk-YiFEO_Ridw6v4z4ABPkiJvWSoeM_gUiNXRs1JZbCqMuVPHAL6iwMb9TTj6Tvu6-XOkLRhhqNQqLFvVU26NP042o30K9Ilk_CWP978sRy7isjuxb_EthQr64wj0TE8TKEBfTubJANRmUCBFBgQ7vgO0lrb-ySSFhXoO_o-870HeAQ7Mefoi6CWaTmH3mbZCP3P2IXz4obWSfWjF8MgB64wiYeAyCVB2Jvg2Hfv3mkP2hasvsUlLYmcFaa9NrRr-PFy_HTOyMQdQYNddyNCjoMu6XWL2_RU8jYazgmxGLvEY-KwvAjDxApWyivBT2u2f5_0-76RR-xggzv5ehNL29FowG3YrZEYbxNJjeaA1uuFWDV_vzKSyRQ-W0Qq0eBWxe6ReK2TKAPixAvUUoYvAv4LXuRGojB-v1MW4F86-w_QAXr1f75yj_FSoglrR4cI") }}</p>
+  <p>{{ _("<a href='{url}' download>DAC Report - September 2019</a>").format(url = "https://storage.cloud.google.com/census-ci-craftcms/assets/ONS-Census-Survey-2021-DAC-Accessibility-Report-Sept-2019.pdf") }}</p>
 
   <p>{{ _("You can read the last full accessibility test report from 28 June 2019.") }}</p>
 
-  <p>{{ _("<a href='{url}' download>DAC Report - June 2019</a>").format(url = "https://00e9e64bac9206c97891d55b342ec14fcbd060bd86f5fcfd7a-apidata.googleusercontent.com/download/storage/v1/b/census-ci-craftcms/o/assets%2FONS-Census-Survey-2021-DAC-Accessibility-Report-June-2019.pdf.pdf?qk=AD5uMEtDN_n0XefOe88HcOOip9fwE_rTOJzdgHBiXX3yCjLzxfhasUlTdMnrUKXhaoFi65ZEh6AVlZ84RZiVg3yldO9vX8pTWV1wBI-01hhi8i6opHB6AbJFO-kKe2YAECZr8z6yxdyk53WLMohu-_ek0QRYZ4W6SwxLo9wLz-EybNJg1m9Mt2nUeW6TXLR0YA_-NGyqC0A6EodtMYEtyVTD_XvLijY0i0uKybP51yYf-FL8nS1skl5ZGK7U-Pc2I_Mg6TpMKyanFdiLDMHbWh8rKepXltl0WFbShkEzE2Jg9mXmhlZ2jRPUZRF_dhL2z-t9wUGCqcSZoO4lqRcfCA7aSlotZ-gKZ3bxtKT2Ldu-U6obP1dZ2uWYKWS4QZOMcJjFP6NpBx2_b15bF87QjuToFjQqYh1jJ4CdeXCdT_SsiS12T8AdFjEhp6J58Qdxv4zLCJug782Gr39y2d95s54ASi6hpKY0YFclwh-IxpJ-XKypID5gmw_ZJy6OYKRnsKkPw11DXsBACGTmxqP3awuhJvFcIFWzJepLV7anjNnII2t0OlrXMVsR3pVQGCKels3KIvIHllzg-yh_9aU51pRBaoEdxiLGgWvKnptQXAblsAEiiTCFSfQX-0mXIDErIgJnsBmsvjV9zkajaHa4xHZCNHJ3SYC67ATe3T2J3_rc-kuZAsi1Nfiyrf3FrH_LkAYDVY00lsnyodGFuHgX-it5S-jhqZk0Zmt01f2X0a_kRWGy2NPl8yb_hOWFH_8iVySv9mgLsiIK1KyXV5NYkU02sDuK5QvNWmNPf6ok2_VtYPy8jIJSECxxlExHUCBWXSiUKK4lzsqWGQTsg2Gr3qNT-0g-oVfwhoApJzkaBU7Z9ndz4FJYAiuJzunS_nt3-d33F0OH27Kz") }}</p>
+  <p>{{ _("<a href='{url}' download>DAC Report - June 2019</a>").format(url = "https://storage.cloud.google.com/census-ci-craftcms/assets/ONS-Census-Survey-2021-DAC-Accessibility-Report-June-2019.pdf") }}</p>
```

Comment:
This link is still broken.

Reply:
Yeah it will be fixed soon. I just need the file to be renamed on the cms for the link to work